addon/store.js

```diff
@@ -46,45 +46,39 @@ var Store = Ember.Object.extend({
       this.set("array", {});
     },
     clear(type) {
-        if(type === undefined) {
+        if (type === undefined) {
             this.reset();
         }
+
         delete this.get("identityMap")[type];
         arrayForType(type, this).clear();
-
-        run.once(() => {
```

Comment:
Without this I believe the benchmark app still worked just fine (something like 126 - 148ms per 100 pushed into a store w/ 4K already). Seems like this was a mistake /misunderstanding on my part when I was trying to make the updateFilters even better but after looking at the commit below it seems that high level test would still pass just the same w/ or without this run.once

https://github.com/toranb/ember-cli-simple-store/commit/0891cded65421417696ea1b8655218fb98656b78